### DISCUSSION TITLE
Proposal: reorganisation of documents

### DISCUSSION THREAD

**Discussion by PsiACE** [databendlabs/databend#3694] [Ideas]
## Background

At https://github.com/datafuselabs/databend/issues/2753, we have a proposal for moving the docs directory.

## Purposes

- Easier to find documentation for newcomers
- Convenient collection of RFCs

## Plan

- Move the developer documentation to the docs directory. (rfcs / contributing)
- Remove the databend directory. (I don't think we will have many websites or maintain a complex node project in this repo)

```
- docs
  - rfcs
  - contributing
- website
  - src
  - docs
    - dev -> link to /docs
    - user
    - overview
```

## Credits

@Xuanwo Plan for this reorganisation

**Comment by bohutang** [comment]
How about link `- docs` to `website/src/docs/dev`?
